grails-datastore-gorm-hibernate5/src/test/groovy/grails/gorm/tests/events/UpdatePropertyInEventListenerSpec.groovy

```diff
@@ -69,6 +69,11 @@ class UpdatePropertyInEventListenerSpec extends Specification {
 class User {
     String username
     String password
+
+    static mapping = {
+        table '`user`'
```

Comment:
Is this because of the h2 change?

Reply:
It is, h2 is stricter about reserved words in 2.x.  We were using 1.4.200, which is 5 years old.  I had seen this before on a coding assignment, where I didn't specify which database applications should use and h2 throws this unexpected curveball.

Reply:
There is not a location to collect these, that I am aware of.  I started one in https://drive.google.com/drive/folders/1gqA_E_XH8694kKmyGLti-oO9RcwCXiKq